### DISCUSSION TITLE
Would it be possible to support temporary file extensions for files being downloaded?

### DISCUSSION THREAD

**Discussion by masterjg** [rogerfar/rdt-client#487] [Ideas]
My use case is:
I have Jellyfin and with rdt client download video file to jellyfin videos folder. Jellyfin immediately picks the video there and adds it to the library.

I'd like to propose option to enable temporary file extensions for files that are in progress...

like for example:
video.mkv.tmp or something similar

**Comment by rogerfar** [comment]
Are you not using something like Radarr or Sonarr to manage this part?
